src/provider/amino/utils.ts

```diff
@@ -0,0 +1,200 @@
+import { NumberUtils, TypeUtils } from "@carbon-sdk/util";
+import { SimpleMap } from "@carbon-sdk/util/type";
+import { AminoConverter } from "@cosmjs/stargate";
+import BigNumber from "bignumber.js";
+import Long from "long";
+
+export enum AminoTypes {
```

Comment:
We need to rename this variable
https://github.com/cosmos/cosmjs/blob/main/packages/stargate/src/aminotypes.ts